src/equations/compressible_euler_3d.jl

```diff
@@ -343,11 +343,11 @@ Should be used together with [`TreeMesh`](@ref).
                                               equations::CompressibleEulerEquations3D)
     # get the appropriate normal vector from the orientation
     if orientation == 1
-        normal_direction = SVector(1.0, 0.0, 0.0)
+        normal_direction = SVector(1, 0, 0)
     elseif orientation == 2
-        normal_direction = SVector(0.0, 1.0, 0.0)
+        normal_direction = SVector(0, 1, 0)
     else # orientation == 3
-        normal_direction = SVector(0.0, 0.0, 1.0)
+        normal_direction = SVector(0, 0, 1)
     end
```

Comment:
This one is tricky. We normalize the `normal_direction` in line 290 above. By default, the `norm` of a vector of integers is a `Float64`. Thus, we should probably use `Float32` here.